pod-operation/src/state_machine.rs

```diff
@@ -221,6 +216,11 @@ impl StateMachine {
 		}
 
 		if self.downstream_pressure_transducer.read_pressure() < MIN_PRESSURE {
+			self.io
+				.of("/control-station")
+				.unwrap()
+				.emit("fault", "Low pressure detected.")
```

Comment:
Chore [repeating]: could we include the current value and the threshold as part of the reason messages, e.g. value A exceeded value B?